bitnami/redis/templates/redis-master-statefulset.yaml

```diff
@@ -219,8 +219,10 @@ spec:
               mountPath: /opt/bitnami/redis/secrets/
             {{- end }}
             - name: redis-data
-              mountPath: {{ .Values.master.persistence.path }}
-              subPath: {{ .Values.master.persistence.subPath }}
+              mountPath: {{ .Values.master.persistence.path | quote }}
+              {{- if .Values.master.persistence.subPath }}
```

Comment:
Why is this `if` needed?

Reply:
Without, the subPath will be rendered empty. Also note the quote for the mountPath.